packages/theme/lang/en.js

```diff
@@ -343,6 +343,10 @@ export default {
       change_to_grid_view: 'Change to grid view',
       change_to_list_view: 'Change to list view'
     },
+    content_page: {
+      not_found: 'Content page not found',
+      try_again_with_different_url: 'Try again with different URL'
```

Comment:
Hm, I'm not fully happy with the error message, it feels overly technical. A regular user doesn't even know that there's something like 'content page' ;)
How about

```suggestion
      not_found: 'Page not found',
      try_again_with_different_url: 'The URL you entered is not valid'
```
or something similar?